install_deps.cmake

```diff
@@ -113,5 +113,5 @@ cget(init ${TOOLCHAIN_FLAG} -DCMAKE_INSTALL_RPATH=${PREFIX}/lib ${PARSE_UNPARSED
 cget(ignore pcre)
 
 # Install dependencies
-cget(install -U pfultz2/rocm-recipes)
+cget(install -U ROCmSoftwarePlatform/rocm-recipes)
```

Comment:
the repository is now located under ROCMSoftwarePlatform, the old path still works but is not guaranteed in the future, so we have updated it.